main.go

```diff
@@ -187,6 +189,9 @@ func main() {
 	if FeatureSearchSettings != "" && FeatureSearchSettings == "true" {
 		util.SetFeatureSearchSettings(true)
 	}
+	if FeatureSynonyms == "true" {
```

Comment:
Also, add a check for an empty string.

Reply:
this is not needed. We're directly comparing a string. So 
`"" == "true"` would result to `false`, which is what we wanted